test/soca/testinput/insitu_GLIDER_2021063006.json

```diff
@@ -0,0 +1,12 @@
+{
+    "data_format": "subpfl",
+    "subsets": "SUBPFL",
+    "source": "NCEP data tank",
+    "data_type": "glider",
+    "cycle_type": "gdas",
+    "cycle_datetime": "2021063006",
+    "dump_directory": "/scratch1/NCEPDEV/stmp2/Edward.Givelberg/RUNDIRS/GFSv17-3DVAR-C384mx025/prepoceanobs.114138",
+    "ioda_directory": "/scratch1/NCEPDEV/stmp2/Edward.Givelberg/RUNDIRS/GFSv17-3DVAR-C384mx025/prepoceanobs.114138",
```

Comment:
same ... I won't comment on the other yaml with the same issues, but they need to be fixed as well.